src/components/Navbar.tsx

```diff
@@ -0,0 +1,35 @@
+import { NavLink } from 'react-router-dom';
+import classNames from 'classnames';
+
+export const Navbar = () => (
+  <nav
+    data-cy="nav"
+    className="navbar is-fixed-top has-shadow"
+    role="navigation"
+    aria-label="main navigation"
+  >
+    <div className="container">
+      <div className="navbar-brand">
+        <NavLink
+          className={({ isActive }) => classNames(
+            'navbar-item',
+            { 'has-background-grey-lighter': isActive },
+          )}
+          to="/"
+        >
+          Home
+        </NavLink>
```

Comment:
Create component for `NavLink` - smth like `NavigationLink`. you repeat almost of the logic in both links. All that's different is `to` path and name of the link - make them props of your new component